src/reducers/computeRenderedRows.js

```diff
@@ -145,6 +145,23 @@ function calculateRenderedRowRange(state, scrollAnchor) {
     rowIdx += step;
   }
 
+  /* Handle the case where we rows have shrunk and we don't have enough content
+     between our start scroll anchor and the end of the table to fill the available space.
+     In this case process earlier rows as needed and act as if we've scrolled to the last row.
+   */
+  let forceScrollToLastRow = false;
+  if (totalHeight < maxAvailableHeight && rowIdx === rowsCount && lastIndex === undefined) {
```

Comment:
why is the `lastIndex` undefined check required?

Reply:
if we're anchoring to the lastIndex then we've already been stepping from the last index row upwards towards the top of the viewport.  If we didn't find enough row content, then stepping upwards more won't help.

A follow up question would be whether we need to step down to rows below the last index in that case.  I don't think so, but if you can come up with a case to try, I'm happy to test it out.